libbeat/docs/shared-ssl-config.asciidoc

```diff
@@ -211,157 +339,234 @@ key: |
 authority (CA) and also verifies that the server's hostname (or IP address)
 matches the names identified within the certificate. If the Subject Alternative
 Name is empty, it returns an error.
- * `certificate`, which verifies that the provided certificate is signed by a
+
+`certificate`::
+Verifies that the provided certificate is signed by a
 trusted authority (CA), but does not perform any hostname verification.
- * `none`, which performs _no verification_ of the server's certificate. This
+
+`none`::
+Performs _no verification_ of the server's certificate. This
 mode disables many of the security benefits of SSL/TLS and should only be used
-after very careful consideration. It is primarily intended as a temporary
+after cautious consideration. It is primarily intended as a temporary
 diagnostic mechanism when attempting to resolve TLS errors; its use in
 production environments is strongly discouraged.
-
++
 The default value is `full`.
 
-[float]
-==== `cipher_suites`
+[discrete]
+[[ssl-server-config]]
+=== Server configuration options
 
-The list of cipher suites to use. The first entry has the highest priority.
-If this option is omitted, the Go crypto library's https://golang.org/pkg/crypto/tls/[default suites]
-are used (recommended). Note that TLS 1.3 cipher suites are not
-individually configurable in Go, so they are not included in this list.
+You can specify the following options in the `ssl` section of the +{beatname_lc}.yml+ config file.
 
-// tag::cipher_suites[]
-The following cipher suites are available:
-
-[options="header"]
-|===
-| Cypher | Notes
-| ECDHE-ECDSA-AES-128-CBC-SHA |
-| ECDHE-ECDSA-AES-128-CBC-SHA256 | TLS 1.2 only. Disabled by default.
-| ECDHE-ECDSA-AES-128-GCM-SHA256 | TLS 1.2 only.
-| ECDHE-ECDSA-AES-256-CBC-SHA |
-| ECDHE-ECDSA-AES-256-GCM-SHA384 | TLS 1.2 only.
-| ECDHE-ECDSA-CHACHA20-POLY1305 | TLS 1.2 only.
-| ECDHE-ECDSA-RC4-128-SHA | Disabled by default. RC4 not recommended.
-| ECDHE-RSA-3DES-CBC3-SHA |
-| ECDHE-RSA-AES-128-CBC-SHA |
-| ECDHE-RSA-AES-128-CBC-SHA256 | TLS 1.2 only. Disabled by default.
-| ECDHE-RSA-AES-128-GCM-SHA256 | TLS 1.2 only.
-| ECDHE-RSA-AES-256-CBC-SHA |
-| ECDHE-RSA-AES-256-GCM-SHA384 | TLS 1.2 only.
-| ECDHE-RSA-CHACHA20-POLY1205 | TLS 1.2 only.
-| ECDHE-RSA-RC4-128-SHA | Disabled by default. RC4 not recommended.
-| RSA-3DES-CBC3-SHA |
-| RSA-AES-128-CBC-SHA |
-| RSA-AES-128-CBC-SHA256 | TLS 1.2 only. Disabled by default.
-| RSA-AES-128-GCM-SHA256 | TLS 1.2 only.
-| RSA-AES-256-CBC-SHA |
-| RSA-AES-256-GCM-SHA384 | TLS 1.2 only.
-| RSA-RC4-128-SHA | Disabled by default. RC4 not recommended.
-|===
+[float]
+[[server-certificate-authorities]]
+==== `certificate_authorities`
 
-Here is a list of acronyms used in defining the cipher suites:
+The list of root certificates for server verifications is only required if 
+`client_authentication` is configured. If `certificate_authorities` is empty or not set, and
+`client_authentication` is not configured, the system keystore is used.
```

Comment:
s/client_authentication is not configured/client_authentication is not configured/